frontend/src/components/pages/CreatorProfile/CreatorPresentations.tsx

```diff
@@ -0,0 +1,113 @@
+import { Divider, Flex, Text } from "@chakra-ui/react";
+import React from "react";
+import { BiFileBlank } from "react-icons/bi";
+import { BsBook } from "react-icons/bs";
+import { IoBrushOutline } from "react-icons/io5";
+
+import Carousel, { Presentations } from "./Carousel";
+
+interface CreatorPresentationsProps {
+  presentations: Presentations[];
+}
+
+const additionalInfoNames = [
+  "Preferred Grade Level(s)",
+  "Preferred Audience Size",
+  "Offered Locations",
+  "Languages Offered",
+  "AV Setup",
+  "Remote Option?",
+  "Book Sale/Signing Session After?",
+  "Price",
+];
+
+const additionalInfoFields = [
+  "gradeLevels",
+  "audienceSize",
+  "locations",
+  "languages",
+  "avSetup",
+  "remote",
+  "saleOrSigning",
+  "price",
+];
+
+const CreatorPresentations = ({
+  presentations,
+}: CreatorPresentationsProps): React.ReactElement => {
+  return (
+    <div>
+      <Text fontSize="30px" fontWeight={700}>
+        Presentations
+      </Text>
+      {presentations.map((pres, idx) => {
+        return (
+          <Flex key={idx} direction="column" mt="24px">
+            <Flex flexWrap="wrap" alignItems="center">
+              <Flex
+                backgroundColor="#E2E8F0"
+                borderRadius="100px"
+                height="40px"
+                width="40px"
+                alignItems="center"
+                justifyContent="center"
+                mr="16px"
+              >
+                {(pres.name === "Readings" && <BsBook />) ||
```

Comment:
nit: For contsants like "Readings" or "Workshops", use a variable like READINGS_NAME to store them